### ISSUE TITLE
Unimplemented colors

### ISSUE THREAD

**Issue by Psyhackological** [Fingel/cosmic-themes-org-py#3]
# Hi! 👋🏼 

Thank you for your effort and such awesome idea and website!

## The issue
![image](https://github.com/user-attachments/assets/4f758569-f978-492a-bf23-7cd0496c4fc1)

I'm just wondering how can I implement those above.

### 🟢 Green 
![image](https://github.com/user-attachments/assets/5decdcb8-2cd1-4b9d-8a0e-629ff43dd8c6)

### 🔴 Red 
![image](https://github.com/user-attachments/assets/7f08e10b-c18b-4147-8640-45e2754f68e4)

### ⚫ Black
![image](https://github.com/user-attachments/assets/2296bac0-cff7-476d-ada5-1cf3681599a6)

## Example

![image](https://github.com/user-attachments/assets/a3ec8f5c-1e39-4ab4-b034-e71070c814a2)

This is my [Factorio](https://factorio.com/) palette.

For now I think it's up to manual work as I don't see any option to configure these in Cosmic.

**Comment by Fingel** [comment]
Hello! 

First I want to say I love your theme, I think it's the best one on the site and I'm using it myself!

As for the colors, you are correct. If you want to set those you will need to manually edit the theme's .ron file as many of the colors are derived from the base palette of the light and dark themes.

Maybe in the future when cosmic-settings has stabilized and is out of beta it would be possible to create an interface to edit these other colors on the site itself.

Does that answer the question?

**Comment by Psyhackological** [comment]
Thanks! I'm glad you liked it so much! :orange_heart:

> I just stopped playing Factorio btw.

I see, so it is not yet implemented into cosmic theme itself, right? I did not see these colors used anywhere. Red and green are probably for some errors and success I'm guessing. Black idk.

**Comment by Fingel** [comment]
Yes that is correct. The colors might be used in some apps, it’s possible none have used them yet. I haven’t seen those colors either. On Aug 30, 2024, at 2:47 PM, Konrad Konieczny ***@***.***> wrote:﻿
Thanks! I'm glad you liked it so much! 🧡

I just stopped playing Factorio btw.

I see, so it is not yet implemented into cosmic theme itself, right? I did not see these colors used anywhere. Red and green are probably for some errors and success I'm guessing. Black idk.
